lib/constants.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @enum {number}
+ */
+const DEFAULT_LEVELS = {
+  trace: 10,
+  debug: 20,
+  info: 30,
+  warn: 40,
+  error: 50,
+  fatal: 60
+}
+
+/**
+ * @enum {"ASC" | "DESC"}
+ */
+const SORTING_ORDER = {
```

Comment:
These jsdoc blocks are missing the documentation part.

Reply:
Done